packages/amplify-provider-awscloudformation/src/utils/admin-helpers.ts

```diff
@@ -0,0 +1,56 @@
+import { stateManager, $TSContext } from 'amplify-cli-core';
+import aws from 'aws-sdk';
+import _ from 'lodash';
+import fetch from 'node-fetch';
+
+export const originUrl = 'https://www.dracarys.app';
```

Comment:
url handling 4th place. No region here either, I guess it would be needed here as well

Reply:
This will need to change to actual regional URL's. cc @jhockett

Reply:
Changed URL's.